featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -761,29 +762,25 @@ def parent_child(request):
 def test_empty_child_dataframe(parent_child):
     parent_df, child_df = parent_child
     if not isinstance(parent_df, pd.DataFrame):
-        parent_vtypes = {
-            'id': variable_types.Index
-        }
         child_vtypes = {
-            'id': variable_types.Index,
-            'parent_id': variable_types.Numeric,
-            'time_index': variable_types.Datetime,
-            'value': variable_types.Numeric,
-            'cat': variable_types.Categorical
+            'parent_id': ww.list_logical_types.Double,
+            'time_index': ww.list_logical_types.Datetime,
+            'value': ww.list_logical_types.Double,
+            'cat': ww.list_logical_types.Categorical
         }
     else:
-        parent_vtypes = None
         child_vtypes = None
     es = ft.EntitySet(id="blah")
-    es.entity_from_dataframe(entity_id="parent",
-                             dataframe=parent_df,
-                             index="id",
-                             variable_types=parent_vtypes)
-    es.entity_from_dataframe(entity_id="child",
-                             dataframe=child_df,
-                             index="id",
-                             time_index="time_index",
-                             variable_types=child_vtypes)
+    es.add_dataframe(dataframe_name="parent",
+                     dataframe=parent_df,
+                     index="id",
+                     semantic_tags={'id': 'index'})
```

Comment:
I don't think we want to manually add the index semantic tag - that should happen automatically.

Reply:
removed